components/sections/call_to_action/variant_b.tsx

```diff
@@ -1,107 +1,86 @@
-                    : `Go to ${logoLink(logo)}`
-                }
-                className="mb-6 inline-block rounded p-3"
-                href={logoLink(logo)}
-              >
-                <Image
-                  src={urlFor(logo?.image)}
-                  width={56}
-                  height={56}
-                  alt={logo?.alt ?? "callToAction-logo"}
-                />
-              </Link>
-            )}
-            <h1 className="font-heading mb-4 text-4xl font-bold md:text-5xl">
-              {title}
-            </h1>
-            <p className="mb-6 text-gray-700">{text}</p>
-            {form?.fields && (
-              <WebriQForm
-                method="POST"
-                data-form-id={form?.id}
-                name="Calltoaction-VariantB-Form"
-                className="flex flex-wrap items-center justify-center"
-                data-thankyou-url={thankYouPageLink(form?.thankYouPage)}
-                scriptsrc="https://pagebuilderforms.webriq.com/js/initReactForms"
-              >
-                {form?.fields
-                  ?.slice(0, 2)
-                  ?.map((field) => (
-                    <input
-                      key={field?._key}
-                      aria-label={field?.placeholder ?? field?.name}
-                      className="mb-3 w-full rounded bg-white px-4 py-2 leading-loose md:mb-0 md:mr-4 md:w-auto"
-                      type={
-                        field?.type === "inputEmail"
-                          ? "email"
-                          : field?.type === "inputPassword"
-                          ? "password"
-                          : field?.type === "inputNumber"
-                          ? "number"
-                          : "text"
-                      }
-                      placeholder={field?.placeholder}
-                      name={field?.name}
-                      required={field?.isRequired}
-                    />
-                  ))}
-                <div>
-                  <div className="webriq-recaptcha" />
-                </div>
-                {form?.buttonLabel && (
-                  <button
-                    aria-label={
-                      form?.buttonLabel ?? "Call to action form submit button"
+    <section className="py-20 bg-gray-50">
+      <Container maxWidth={576} className="text-center">
+        {logo?.image && (
+          <Link
+            aria-label={
+              logoLink(logo) === "/"
+                ? "Go to home page"
+                : `Go to ${logoLink(logo)}`
+            }
+            className="inline-block p-3 mb-6 rounded"
+            href={logoLink(logo)}
+          >
+            <Image
+              src={urlFor(logo?.image)}
+              width={56}
+              height={56}
+              alt={logo?.alt ?? "callToAction-logo"}
+            />
+          </Link>
+        )}
+        {title && <Heading className="mb-4">{title}</Heading>}
+        <Text className="mb-6">{text}</Text>
```

Comment:
Just like the Title and other elements, kindly update the Text element to be returned only when it is defined/there is a value.